_includes/mobile-pay-payments-capture.md

```diff
@@ -0,0 +1,63 @@
+## Capture
```

Comment:
Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)